bin/commands/runs.js

```diff
@@ -63,6 +60,9 @@ module.exports = function run(args) {
       // accept the number of parallels
       utils.setParallels(bsConfig, args, specFiles.length);
 
+      // set Local Mode (on-demand/ always-on)
+      utils.setLocalMode(bsConfig, args);
```

Comment:
We depend on local mode for setting up local_identifier, I can still see local identifier getting set above.